discovery-provider/src/tasks/index.py

```diff
@@ -106,9 +106,11 @@ def get_latest_block(db):
 
 def update_latest_block_redis():
     latest_block_from_chain = update_task.web3.eth.getBlock('latest', True)
+    default_indexing_interval_seconds = int(update_task.shared_config["discprov"]["block_processing_interval_sec"])
     redis = update_task.redis
-    redis.set(latest_block_redis_key, latest_block_from_chain.number)
-    redis.set(latest_block_hash_redis_key, latest_block_from_chain.hash.hex())
+    # these keys have a TTL which is the indexing interval
+    redis.set(latest_block_redis_key, latest_block_from_chain.number, ex=default_indexing_interval_seconds)
```

Comment:
is there a possible race cond. between the celery task actually running at `default_indexing_interval_seconds` and this getting expired at `default_indexing_interval_seconds`?

Reply:
since this is not used anywhere for indexing related tasks i'm not super worried. indexing actually does a web3 get latest call before each task anyway. plus this is a set while the health check override is setnx so it'll prefer the health check set value and it won't override

Reply:
i was going to suggest we make this last exactly 1s longer or something funky like that but based on the above idt its a big deal